### PR TITLE
CSS: Add dark mode

### PR DESCRIPTION
**Pull request by cal0pteryx**
image

@pulkomandy 
@linkmauve

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -112,3 +102,53 @@ a.button:hover {
 	border-radius: 5px;
 	background-color: #f1f1f1;
 }
+
+@media (prefers-color-scheme: dark) {
+	body {
+        background-color: #161616;
+        color: #ccc;
```
Comment (linkmauve) [inline style.css]: You probably want to use tabs here.

**Comment by linkmauve** [comment]
Also the green on green in “complete”, and yellow on yellow in “partial” have a quite poor contrast on my screen, you may be interested in https://wave.webaim.org/ to evaluate accessibility for those kind of pages.

Aside from that, that looks great!

**Comment by cal0pteryx** [comment]
> Also the green on green in “complete”, and yellow on yellow in “partial” have a quite poor contrast on my screen, you may be interested in https://wave.webaim.org/ to evaluate accessibility for those kind of pages.
> 
> Aside from that, that looks great!

You are right! I increased the contrast for both 'Complete' and 'Partial' labels, and for other elements as well.

**Comment by pulkomandy** [comment]
Thanks!

**Pull request merged**